src/navigation/context.py

```diff
@@ -29,6 +30,20 @@ class Gate:
     post1: np.ndarray
     post2: np.ndarray
 
+    def getPostShape(self):
+        """
+        Creates a circular path of RADIUS around each post for checking intersection with our path
+        """
+
+        # Declare radius to 0.5 meters
+        RADIUS = 0.5
+
+        # Find circle of both posts
+        post1_shape = Point(self.post1[0], self.post1[1]).buffer(RADIUS)
```

Comment:
apparently shapely constructors can take in numpy arrays, so these can be changed to `Point(self.post1[:2]).buffer(RADIUS)` (not even that sweaty)